pysat/tests/test_utils_coords.py

```diff
@@ -68,31 +73,48 @@ def test_update_longitude(self, name):
 
         assert np.all(self.py_inst.data['longitude'] < 180.0)
         assert np.all(self.py_inst.data['longitude'] >= -180.0)
+        return
 
     def test_bad_lon_name_update_longitude(self):
         """Test update_longitude with a bad longitude name"""
 
         self.py_inst = pysat.Instrument(platform='pysat', name="testing")
         self.py_inst.load(date=self.inst_time)
 
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError) as verr:
             coords.update_longitude(self.py_inst, lon_name="not longitude")
 
-    #########################
-    # calc_solar_local_time
+        assert str(verr).find("unknown longitude variable name") >= 0
+        return
+
+
+class TestCalcSLT():
+    """Unit tests for `calc_solar_local_time` and integration tests for
+    `update_longitude`.
+    """
```

Comment:
too long!

Reply:
But it needs all this info!  Technically these can be two lines long, and since this one doesn't get printed out as an error message (unlike the test methods) the stakes are lower.  Open to suggestions that imparts all the same info but is shorter.